.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # check issue and pull request once every day
+    - cron: '25 11 * * *'
+
+jobs:
+  invalid-stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
Please update the phrases according to https://github.com/open-mmlab/mmdetection/pull/8002